build.sh

```
@@ -1,43 +1,6 @@
-            echo "Can not find nodejs. Please try again."
-        fi
-    fi
-fi
-
-if ! command -v vsce; then
-    npm install -g vsce
-fi
+# Wrapper script for backward compatibility
+# Use tasks.sh directly for more options: ./tasks.sh --help
 
 cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)